probs/probs.go

```diff
@@ -15,6 +15,7 @@ const (
 	UnknownHostProblem    = ProblemType("urn:acme:error:unknownHost")
 	RateLimitedProblem    = ProblemType("urn:acme:error:rateLimited")
 	BadNonceProblem       = ProblemType("urn:acme:error:badNonce")
+	InvalidEmailProblem   = ProblemType("urn:acme:error:invalidEmail")
```

Comment:
Consensus was to register error messages with the spec if they are under `urn:acme:error:` namespace. I don't see any changes (or PRs) in the spec...